### PR TITLE
Created mBean with OSGi

### PR DESCRIPTION
**Pull request by FlynnIBM**
#19420

### CODE REVIEW

File: dev/com.ibm.ws.security.authentication.builtin/src/com/ibm/ws/security/authentication/internal/cache/RevokeAuthCacheUsersImpl.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferencePolicyOption;
+
+import com.ibm.ws.security.authentication.cache.AuthCacheService;
+import com.ibm.ws.security.authentication.cache.RevokeAuthCacheUsers;
+
+/**
+ *
```
Comment (mmerali96) [inline dev/com.ibm.ws.security.authentication.builtin/src/com/ibm/ws/security/authentication/internal/cache/RevokeAuthCacheUsersImpl.java]: Add javadoc for external mbean

**Comment by FlynnIBM** [comment]
#build

**Comment by FlynnIBM** [comment]
#run-libby-bot

**Comment by FlynnIBM** [comment]
RTC Results had only existing unrelated issues.

**Pull request merged**